test/e2e/api_inheritance/api_inheritance_test.go

```diff
@@ -180,6 +180,13 @@ func TestAPIInheritance(t *testing.T) {
 		}
 
 		// Update target workspace to inherit from source
+		targetWorkspace, err = kcpAdminClient.TenancyV1alpha1().Workspaces().Get(ctx, targetWorkspace.GetName(), metav1.GetOptions{})
```

Comment:
I'm curious why you've added this?

Reply:
Since the workspace controller is running now, it appears to update the workspace resource created earlier in this function.  So we need to pull the updated version.